src/common/components/registerAccountModal/registerAccountModal.tpl.html

```diff
@@ -1,34 +1,74 @@
-<div ng-switch="$ctrl.state">
+<div class="modal-header">
+  <button type="button" class="close" aria-label="Close" ng-click="$ctrl.onCancel()">
+    <span aria-hidden="true">×</span>
+  </button>
+  <div class="container-fluid">
+    <div class="row">
+      <div class="col-xs-12">
+        <div ng-if="$ctrl.state !== 'sign-up'">
+          <h3 ng-if="$ctrl.firstName" translate>Welcome back, {{$ctrl.firstName}}!</h3>
+          <h3 ng-if="!$ctrl.firstName" translate>Welcome back!</h3>
+        </div>
+        <div ng-if="$ctrl.state === 'sign-up'" class="signup-header">
+          <img src="{{$ctrl.imgDomain}}/assets/img/cru-logo.png" alt="Cru logo" />
+          <span>+</span>
+          <img src="{{$ctrl.imgDomain}}/assets/img/okta.png" alt="Okta logo" />
+        </div>
+        <div ng-if="$ctrl.state === 'sign-in'" class="section">
+          <ul>
+            <li translate>Give to thousands of opportunities.</li>
+            <li translate>View giving history (online and offline).</li>
+            <li translate>Access your tax receipts and forms.</li>
+          </ul>
+        </div>
+        <div ng-if="$ctrl.state === 'contact-info'" class="section paragraph" translate>
+          Your email address is verified. Please provide your contact information so that we can send you your giving receipts.
+        </div>
+        <div class="section cart" ng-if="$ctrl.cartCount > 0">
+          <span ng-if="$ctrl.cartCount === 1" translate>1 gift is currently in your cart.</span>
+          <span ng-if="$ctrl.cartCount > 1">{{$ctrl.cartCount}} <span translate>gifts are currently in your cart.</span></span>
+        </div>
+      </div>
+    </div>
+  </div>
+</div>
+<div class="modal-body" ng-switch="$ctrl.state">
   <sign-in-modal ng-switch-when="sign-in"
-                 modal-title="$ctrl.modalTitle"
-                 on-state-change="$ctrl.stateChanged(state)"
-                 on-success="$ctrl.onIdentitySuccess()">
-  </sign-in-modal>
+    modal-title="$ctrl.modalTitle"
+    on-state-change="$ctrl.stateChanged(state)"
+    on-success="$ctrl.onIdentitySuccess()"
+    on-failure="$ctrl.onIdentityFailure()"
+  ></sign-in-modal>
   <sign-up-modal ng-switch-when="sign-up"
-                 modal-title="$ctrl.modalTitle"
-                 on-state-change="$ctrl.stateChanged(state)"
-                 on-success="$ctrl.onIdentitySuccess()">
-  </sign-up-modal>
-  <forgot-password-modal ng-switch-when="forgot-password"
-                         modal-title="$ctrl.modalTitle"
-                         on-state-change="$ctrl.stateChanged(state)">
-  </forgot-password-modal>
+    on-state-change="$ctrl.stateChanged(state)"
```

Comment:
One code style we've used historically, and I'd prefer to continue using, is to line up attributes on multi-line html elements. This would mean either keeping the spacing as it was or moving the `ng-switch-when` down a line and in line with these properties.